src/Dapr.Actors.AspNetCore/ActorsEndpointRouteBuilderExtensions.cs

```diff
@@ -3,31 +3,60 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 
-namespace Dapr.Actors.AspNetCore
+namespace Microsoft.AspNetCore.Builder
```

Comment:
It's the common convention to stick these things in the same namespace as built in stuff so it's easy to find. (yes this is OK in this case, but not in general).